README.rst

```diff
@@ -0,0 +1,60 @@
+DPLASMA  is the leading implementation of a dense linear algebra package
+for distributed, accelerated, heterogeneous systems. It is designed to 
+deliver sustainedperformance for distributed systems where each node featuring
+multiplesockets of multicore processors, and if available, accelerators like
+GPUs or Intel Xeon Phi. DPLASMA achieves this objective through the state of
+the art PaRSEC runtime, porting the Parallel Linear Algebra Software for
+Multicore Architectures (PLASMA) algorithms to the distributed memory realm.
+
+.. contents:: Table of Contents
+
+Features
+========
+
+DPLASMA’s feature set includes:
+
+* Linear systems of equations (Cholesky, LU [inc. pivoting, PP], LDL [prototype]).
+* Least squares (QR & LQ).
+* Symmetric eigenvalue problem (Reduction to Band [prototype]).
+* Level 3 Tile BLAS (GEMM, TRSM, TRMM, HEMM/SYMM, HERK/SYRK, HER2K/SYR2k.
+* Covers double real, double complex, single real, and single complex (D, Z, S, C) precisions.
+* Provides ScaLAPACK-compatible interface for matrices in F77 column-major layout
+* Supports: Linux, Windows, Mac OS X, UN*X (depends on MPI, hwloc)
+
+
+History
+=======
+
+DPLASMA started as the distributed version of ICL PLASMA linear algebra package
+implemented on top of the `PaRSEC`_ runtime, a generic framework for architecture
+aware scheduling and management of micro-tasks on distributed many-core heterogeneous
+architectures. For a long period, the DPLASMA library was packaged as part of the PaRSEC
+project. DPLASMA is now an independent project and uses PaRSEC as a dependency.
+
+
+Installation
+============
+
+Please read the `INSTALL.rst`_ file for the software dependencies and the installation
+instructions. In addition, the `PaRSEC wiki`_ might contain additional information.
```

Comment:
Can we put links in here instead of code markdown?
```suggestion
Please read the `INSTALL.rst` file for the software dependencies and the installation
instructions. In addition, the `PaRSEC wiki` might contain additional information.
```